cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -82,4 +93,36 @@ <h4 class="u-mb5">
     </li>
     {% endfor %}
 </ul>
+
+{% if settings.view_more %}
+  {{ _view_more(settings.view_more) }}
+{% endif %}
+
+{% endmacro %}
+
+{# ==========================================================================
+
+   _view_more()
```

Comment:
It already is, it's used in the Hero and FCM. That said, lets get this merged so it's not hanging out forever and we can add a task to refactor it globally.